### PR TITLE
Fix #640: Add documentation for temporary keys

### LINKED ISSUE #640: Add documentation for temporary keys
We implemented a new feature with temporary encryption keys that should be documented in the scope of the base crypto protocol.

### PR DESCRIPTION
**Pull request by petrdvorak**

### CODE REVIEW

**Review by hvge — CHANGES_REQUESTED** [review]
I think we should also update a [End-To-End-Encryption](https://developers.wultra.com/components/powerauth-crypto/1.8.x/documentation/End-To-End-Encryption) section. There's still information that ECDH is computed with static keys. Also there's missing information about `temporaryKeyId` attribute in the payload.

File: docs/End-To-End-Encryption.md [hunk 1]
```diff
@@ -56,17 +59,17 @@ Note that the `APPLICATION_SECRET` constant is in Base64 form, so we need to rei
 
 #### Activation Scope
 
-ECIES in activation scope has following configuration of parameters:
+ECIES in activation scope has the following configuration of parameters:
 
-- `KEY_ENC_PUB` is `KEY_SERVER_PUBLIC` (e.g. key which is unique for each activation)
+- `KEY_ENC_PUB` is a [temporary key](./Temporary-Encryption-Keys.md) with given `TEMP_KEY_ID` identifier fetched from the server associated with a specific application version and activation, and signed with `KEY_SERVER_PUBLIC` (the key which is unique for each activation, to prove it was intended for the activations cope).
```
Comment (hvge) [inline docs/End-To-End-Encryption.md:64]: Signed with `KEY_SERVER_PRIVATE`.

**Review by hvge — CHANGES_REQUESTED** [review]
Just one small fix and it would be OK.

File: docs/End-To-End-Encryption.md (hunk 1 above)
Comment (petrdvorak) [reply docs/End-To-End-Encryption.md]: Thank you!

**Pull request merged**